docs/catalog/cloud/tag-resources.md

```diff
@@ -0,0 +1,34 @@
+---
+version: 1.0
+submitted_by: yelghali
+published_date: tbd
+category: cloud
+tags: 
+- operations
```

Comment:
`operations` is not one of the supported tags for green software patterns: https://patterns.greensoftware.foundation/guide/suggested-tags